darwin/exporter/formats/mask.py

```diff
@@ -512,7 +512,7 @@ def export(
             image = Image.fromarray(mask)
         image.save(outfile)
 
-    with open(output_dir / "class_mapping.csv", "w") as f:
+    with open(output_dir / "class_mapping.csv", "w", newline="") as f:
```

Comment:
In windows, files generated has an extra line in b/w every row, which fails the e2e test as it fails to compare the output and expected files. Found the fix from these links:
- https://stackoverflow.com/a/3191811
- https://stackoverflow.com/a/3348664

Reply:
I think that's a difference between line endings for Windows and Linux/Posix being interpreted different by the writer (`\r\n` vs `\n`), but couldn't swear to it.

Either way, you've solved it.  Nice one.